Gaijinizer patch/CommonEvents_64.txt

```diff
@@ -4,65 +4,73 @@
 
 同世代のサキュバス達の言っていたことが分かります…
 > CONTEXT: CommonEvents_64/43/Dialogue/yuuma
-This way...When you hold you ...
-You can see what the same generation of succubus
-was saying ...#MTLed
+When I'm holding you... like this...
+I can understand what those succubi from my
+generation were talking about.#Ron
 > END STRING
 
 > BEGIN STRING
 この感覚を…
 
 あの時に知っていれば……私は……
 > CONTEXT: CommonEvents_64/47/Dialogue/yuuma
-This feeling...If I knew that time...I ...
-...#MTLed
+This sensation...
+
+If I'd known back then... I...#Ron
 > END STRING
 
 > BEGIN STRING
 やめましょう…♡
 
 今はあなたを抱けることを、素直に喜びます♡♡
 > CONTEXT: CommonEvents_64/52/Dialogue/yuuma
-Let's stop...♡. I'm honestly happy to embrace
-you now#MTLed
+Well let's just leave it at that... ♡
+
+Being able to embrace you makes me geniunely
+happy. ♡♡#Ron
 > END STRING
 
 > BEGIN STRING
 さぁ\\N[1]…♡
 
 一気にいきますよ……♡♡
 > CONTEXT: CommonEvents_64/56/Dialogue/yuuma
-Now \\N[1]...♡. I will go at a stretch...♡♡#MTLed
+Now \\N[1]...♡
+
+I'm going to do it in one big shot... ♡♡#Ron
 > END STRING
 
 > BEGIN STRING
 あ……あひぃ…♡♡
 
 あひいいぃぃぃ♡♡
 > CONTEXT: CommonEvents_64/160/Dialogue/kazuya-0
-Ah...ahnoha...♡♡. yeesss ♡♡#MTLed
+A-... Ahhiii...♡♡
+
+Ahhhiiii. ♡♡#Ron
 > END STRING
 
 > BEGIN STRING
 いい顔ですね…♡\\N[1]。
 
 大丈夫です、ちゃんと加減は制御できていますから…
 > CONTEXT: CommonEvents_64/164/Dialogue/yuuma
-That's a good face...♡ \\N[1].
-It's okay, because you can control it properly
-...#MTLed
+That's a good face... ♡ \\N[1].
+Don't worry, I have this under control...#Ron
 > END STRING
```

Comment:
>_**That's a good face, \\\N[1]... ♡**_

Reply:
The word "exquisite" springs to mind.

Reply:
If that's a good fit with the original, let's go with that, then.
>_**That's an exquisite face....\\\N[1]. ♡**_